src/chat.rs

```diff
@@ -3266,6 +3266,7 @@ pub async fn marknoticed_chat(context: &Context, chat_id: ChatId) -> Result<()>
             .await?;
 
         for chat_id_in_archive in chat_ids_in_archive {
+            start_chat_ephemeral_timers(context, chat_id_in_archive).await?;
```

Comment:
So the `if chat_id.is_archived_link()` branch has two transactions, but this adds N transactions more. Not sure this is a problem, just for the record